addons/portal/views/portal_templates.xml

```diff
@@ -590,12 +593,68 @@
         </t>
     </template>
 
+    <template id="portal_my_users_accesses">
```

Comment:
We could define it in its own file (portal_templates_user_account.py file for example, to have user account management in a file).

Reply:
Do you mean the template only, or should the controller part of company portal access also be in a separate file?